content/en/docs/Writing policies/background.md

```diff
@@ -14,14 +20,12 @@ spec:
   - name: default-deny-ingress
 ```
 
-Background scanning, enabled by default in a `Policy` or `ClusterPolicy` object with the `spec.background` field, allows Kyverno to scan existing resources and find if they match any `validate` rules. If existing resources are found which would violate an existing policy, the background scan notes them in a `ClusterPolicyReport` or a `PolicyReport` object, depending on if the resource is namespaced or not. It does not block any existing resources that match a `validate` rule, even in `enforce` mode. Background scanning is an optional field and defaults to `true`, only taking effect on `validate` rules. It has no effect on either `generate` or `mutate` rules.
 
-By default, background scanning occurs every 15 minutes and is not currently configurable.
+When background scans are enabled for a policy, if existing resources are found which would violate the policy, the background scan reports them using a `ClusterPolicyReport` or a `PolicyReport` object, depending on if the resource is namespaced or not. 
 
-{{% alert title="Note" color="info" %}}
-⚠️  Kyverno does not mutate existing resources to prevent inadvertent changes to workloads.
-Mutate and generate rules are not processed during background scans.
-{{% /alert %}}
+The bachround scan does not block, or otherwise impact, any existing resources that match a `validate` rule, even in `enforce` mode. Background scanning is an optional field and defaults to `true`, only taking effect on `validate` rules. It has no effect on either `generate` or `mutate` rules.
+
+By default, background scanning occurs at start and then every 15 minutes. Currently the scan interval is not configurable.
```

Comment:
"occur at start" is confusing to me. Start of what? Start of creating a policy? Either clarify what that "start" trigger is or omit.

Reply:
Its when Kyverno starts. Clarified!